src/CMakeLists.txt

```diff
@@ -1,5 +1,34 @@
+cmake_minimum_required(VERSION 3.10.0)
+
 include_directories(.)
 
+project(verona-lang CXX)
+
+message(STATUS "Build Type for Verona ${CMAKE_BUILD_TYPE}")
+set(CMAKE_CONFIGURATION_TYPES  Release Debug RelWithDebInfo)
+message(STATUS "Build types ${CMAKE_CONFIGURATION_TYPES}")
+
+if (VERONA_CI_BUILD)
+  # Specify policy to go into child projects.
+  set (CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+  set (SNMALLOC_CI_BUILD ON)
+endif ()
+
+include(../cmake/enable-asserts.cmake)
+
+add_subdirectory(../external/CLI11 ./external/CLI11 EXCLUDE_FROM_ALL)
```

Comment:
Wasn’t there a thing about snmalloc has to live in `src/rt` because we can’t import stuff from parent directories?

If this works here, could we use the same trick to move snmalloc to the top level external?

(at a later point, I’m obviously not suggesting we do this now)

Reply:
I was thinking exactly the same.  I was going to do that as a separate PR.

Reply:
Great